services/app-api/utils/validation/completionSchemas.test.ts

```diff
@@ -69,72 +70,74 @@ describe("Schemas", () => {
 
   const badValidNumberTestCases = ["N/A", "number", "foo"];
 
-  const testNumberSchema = (
+  const testSchema = async (
     schemaToUse: MixedSchema,
-    testCases: Array<string>,
+    testCases: Array<any>,
     expectedReturn: boolean
   ) => {
     for (let testCase of testCases) {
-      let test = schemaToUse.isValidSync(testCase);
-      expect(test).toEqual(expectedReturn);
-    }
-  };
-
-  const testValidNumber = (
-    schemaToUse: MixedSchema,
-    testCases: Array<string | number>,
-    expectedReturn: boolean
-  ) => {
-    for (let testCase of testCases) {
-      let test = schemaToUse.isValidSync(testCase);
+      let test = await schemaToUse.isValid(testCase);
       expect(test).toEqual(expectedReturn);
     }
   };
 
   test("Evaluate Number Schema using number scheme", () => {
-    testNumberSchema(number(), goodNumberTestCases, true);
-    testNumberSchema(number(), badNumberTestCases, false);
+    testSchema(number(), goodNumberTestCases, true);
+    testSchema(number(), badNumberTestCases, false);
```

Comment:
since `testSchema` is now an async function, do we need to `await` these invocations? 
I also think it's ok to keep `isValidSync` and not do async for these tests.